autobahn/twisted/rawsocket.py

```diff
@@ -198,20 +198,20 @@ def dataReceived(self, data):
                 if self.factory.debug:
                     self.log.info("WampRawSocketProtocol: opening handshake received - {0}".format(binascii.b2a_hex(self._handshake_bytes)))
 
-                if ord(self._handshake_bytes[0]) != 0x7f:
+                if ord(self._handshake_bytes[0:1]) != 0x7f:
                     if self.factory.debug:
```

Comment:
It makes some form of sense, if your interpretation of a bytestring is a "series of octets" rather than strings. But still, confusing :(